### PR TITLE
Add Planet logo

### PR DESCRIPTION
**Pull request by digitaltopo**
**Issue:** n/a
**Alexa rank:** [~82.2k](https://www.alexa.com/siteinfo/planet.com)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Adds Planet.com logo

planet

- Hex value is the official brand primary color
- Logo was created from existing SVG (not manually vectorized)
- I work for Planet

### CODE REVIEW

**Comment by PeterShaggyNoble** [comment]
Welcome to Simple Icons, @digitaltopo and thanks for the contribution 👍

Could you provide is with the original, source file you based your SVG on so we can do a quick comparison? I couldn't find that version of the logo in use on your site.

**Comment by digitaltopo** [comment]
Thanks @PeterShaggyNoble This is the "small" version of our logo. I've uploaded the svg file below:
[logo-small.svg.zip](https://github.com/simple-icons/simple-icons/files/4860743/logo-small.svg.zip)

**Comment by digitaltopo** [comment]
Thanks! It is used as the favicon on planet.com, but I don't think we use
it directly anywhere on our marketing website. You can also view it on some
of our apps, ex) Planet.com/explorer (but requires sign up).

**Comment by PeterShaggyNoble** [comment]
Thanks for the info, @digitaltopo. Seeing as https://www.planet.com/explorer/ includes a vector version of the icon on the page that can be accessed even without logging in, let's go with that. Assuming the other @simple-icons/maintainers agree?

**Comment by digitaltopo** [comment]
Sounds good!

File: _data/simple-icons.json [hunk 1]
```diff
@@ -4560,6 +4560,11 @@
             "hex": "F86001",
             "source": "https://www.pjsip.org/favicon.ico"
         },
+        {
+            "title": "Planet",
+            "hex": "009DB1",
+            "source": "https://planet.com"
```
Comment (ericcornelissen) [inline _data/simple-icons.json:4566]: Let's change it and get this merged then! :tada: 

```suggestion
            "source": "https://www.planet.com/explorer/"
```

Comment (PeterShaggyNoble) [reply _data/simple-icons.json]: In the interests of getting this merged, I've committed this suggestion.

**Pull request merged**